packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -4,6 +4,319 @@
 
 import 'package:package_config/package_config.dart';
 
+const String _currentDirectoryScript = r"""
+var _currentDirectory = (function () {
+  var _url;
```

Comment:
uber-nit--I think this could be declared the first place it is assigned, at line 29?

Reply:
Done